CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2019-07-26
```

Comment:
I'm thinking if we shouldn't also include changes for catalogbot env variable in this release. But I won't have time to work on that today :(

Reply:
I think it is fine to do this later in a major version. Here we have a minor version with a new functionality, but refactoring and breaking change will go into a major version. Fine IMO.